Formula/libxcb.rb

```diff
@@ -35,7 +35,7 @@ class Libxcb < Formula
   depends_on "libxdmcp"
 
   on_macos do
-    depends_on "python@3.9" => :build
+    depends_on "python@3.10" => :build
   end
   on_linux do
```

Comment:
Is this still needed? `python@3.{9,10}` no longer depend on `tcl-tk`.

Reply:
Agreed. I cleaned the whole thing up, this should be easier now.